src/test/resources/application-integration-test.properties

```diff
@@ -0,0 +1,21 @@
+server.port=8080
```

Comment:
I think we should try to compose property files, only overriding some of the values between this integration-test file and the test/resources/application.properties file. I think Spring has a way to do that.

Reply:
done